CHANGELOG.md

```diff
@@ -5,6 +5,127 @@ All notable changes to this project will be documented in this file. See
+  - `FluidTypography` add new directive (https://github.com/taiga-family/taiga-ui/pull/8316).
+  - `Progress` add options with DI (https://github.com/taiga-family/taiga-ui/pull/8061).
+  - `Status` add new component (https://github.com/taiga-family/taiga-ui/pull/8057).
+  - `Emails` add pipe (https://github.com/taiga-family/taiga-ui/pull/7596).
+  - `Pulse` add new component (https://github.com/taiga-family/taiga-ui/pull/7544).
+  - `Chevron` add new directive (https://github.com/taiga-family/taiga-ui/pull/7153).
+  - `AvatarOutline` add new directive (https://github.com/taiga-family/taiga-ui/pull/7087).
+  - `NumberFormat` allow dynamic changes (https://github.com/taiga-family/taiga-ui/pull/6856). BREAKING CHANGES: remove
+    `TUI_NUMBER_FORMAT_OBSERVABLE` in favour of `TUI_NUMBER_FORMAT`, `tuiFormatNumber` pipe requires async pipe now.
+  - `Block` add new component (https://github.com/taiga-family/taiga-ui/pull/6892).
+- **@taiga-ui/cdk:**
+  - `Dialogs` refactor creation way (https://github.com/taiga-family/taiga-ui/pull/6660).
+  - `Provide` add new util (https://github.com/taiga-family/taiga-ui/pull/7168).
+  - `TakeUntilDestroyed` add helper (https://github.com/taiga-family/taiga-ui/pull/7381).
+  - `NativeElement` add util (https://github.com/taiga-family/taiga-ui/pull/7393).
+  - `ToArray` add util (https://github.com/taiga-family/taiga-ui/pull/7489).
+  - `DirectiveBinding` / `DirectiveListener` add utils (https://github.com/taiga-family/taiga-ui/pull/7611).
+  - `Control` add signal based abstraction (https://github.com/taiga-family/taiga-ui/pull/7650).
+  - `ThemeColorService` add service (https://github.com/taiga-family/taiga-ui/pull/8240).
+  - `Portal` refactor abstractions (https://github.com/taiga-family/taiga-ui/pull/6692).
+  - Rename and remove some helper instances (https://github.com/taiga-family/taiga-ui/pull/7065).
+  - Scroll controls visible only inside `tui-root` area (https://github.com/taiga-family/taiga-ui/pull/6623).
+- **@taiga-ui/layout:**
+  - `Title`, `BlockDetails`, `Comment` add new components ([#5743](https://github.com/taiga-family/taiga-ui/pull/5743),
+    [#7010](https://github.com/taiga-family/taiga-ui/pull/7010)).
+  - `CardMedium`, `CardLarge` add new components ([#7260](https://github.com/taiga-family/taiga-ui/pull/7260)).
+  - `Cell` add new component ([#5460](https://github.com/taiga-family/taiga-ui/pull/5460),
+    [#7235](https://github.com/taiga-family/taiga-ui/pull/7235)).
+  - `Navigation` implement grid (https://github.com/taiga-family/taiga-ui/pull/8377). Simplify 2nd level aside
+    (https://github.com/taiga-family/taiga-ui/pull/8141).
+- **@taiga-ui/addon-commerce:**
+  - `InputCard` migrate to new `Textfield` ([#7581](https://github.com/taiga-family/taiga-ui/pull/7581)).
+- **@taiga-ui/addon-doc:**
+  - `ThemeSwitcher` add new component (https://github.com/taiga-family/taiga-ui/pull/7966).
+  - `Markdown` add pipe (https://github.com/taiga-family/taiga-ui/pull/6294).
```

Comment:
<img width="688" alt="image" src="https://github.com/user-attachments/assets/d0069676-8985-42d7-94da-e51d3a030320">

Reply:
probably using `markdown-it` utils instead TuiMarkdownPipe?
it is worth replacing this point with something else or removing it

Reply:
just remove it

Reply:
fixed